src/pint/models/astrometry.py

```diff
@@ -535,10 +531,21 @@ def barycentric_radio_freq(self, toas):
         return tbl["freq"] * (1.0 - v_dot_L_array / const.c)
 
     def get_psr_coords(self, epoch=None):
-        """Returns pulsar sky coordinates as an astropy ecliptic coordinates
-        object. Pulsar coordinates will be computed at current coordinates.
+        """Returns pulsar sky coordinates as an astropy ecliptic coordinate instance.
+
+        Parameters
+        ----------
+        epoch: `astropy.time.Time` or Float, optional
+            new epoch for position.  If Float, MJD(TDB) is assumed
+
+        Returns
+        -------
+        position
+        PulsarEcliptic SkyCoord object optionally with proper motion applied
```

Comment:
Should this be indented to render okay? Maybe check the output of `make docs`

Reply:
I overlooked it when updating the docstring, fixed in next commit.